x/rollapp/types/errors.go

```diff
@@ -32,6 +32,7 @@ var (
 	ErrNotFound                       = errorsmod.Register(ModuleName, 1037, "not found")
 	ErrLogic                          = errorsmod.Register(ModuleName, 1038, "internal logic error")
 	ErrInvalidAddress                 = errorsmod.Register(ModuleName, 1040, "invalid address")
+	ErrRollappIDNotEIP155             = errorsmod.Register(ModuleName, 1041, "rollapp-id is not EIP155 compliant")
```

Comment:
there is already ErrInvalidRollappID so this error should wrap it, not be a new one